dashboard/src/shared/Auth.ts

```diff
@@ -46,4 +46,8 @@ export class Auth {
   }
 }
 
-export const axios = Axios.create();
+const requestConfig = {
+  timeout: 5000, // After the timeout the request gets canceled (default no-timeout)
```

Comment:
Installing some larger charts might take longer with Tiller Proxy, we should check to make sure that won't be an issue. cc @andresmgot

Reply:
Yes, I think that 5s is too short. Imagine kubeapps that runs a job to cleanup stuff before deleting the elements. Tiller-proxy doesn't return a response until it receives the response from Tiller (and Tiller doesn't send the response until the deletion is completed). That takes longer than 5s. We should either set a bigger timeout (around even two minutes) or modify the default for Tiller Proxy.

Reply:
Thanks for the info.

2 minutes, which chart can take that long to deploy? Isn't k8s or Nginx going to terminate the connection before then?

Reply:
I'm not sure we've tested it, Kubeapps itself is an example of a chart with hooks that will take a bit longer to install. We should probably look into doing the chart installation more asynchronously.

Reply:
I see, I have noticed that our frontend Nginx gives us only 1 minute before time out, at least connection timeouts. That's why I am going to remove the timeout configuration from this patch and revisit it later https://github.com/kubeapps/kubeapps/issues/619